.github/issue-templates/release.md

```diff
@@ -0,0 +1,59 @@
+# Release {version}
+We are releasing a new version, this issue will keep track of the progress between the first release candidate (20th of each month) to the final release (27th of each month).
+
+After the 20th of each month we start the release process that ends 7 days after, during that period we enter a Feature Freeze. While in the Feature Freeze we do not merge new features or non critical or important bug fixes.  Fixes for bugs created by this release will be prioritized.
+
+For any regression, open a new issue and link to this one.
+
+
+## Before Release - Preparation - 1 business day before the day 20th
+- [x] Create the issue to track the release progress
+- [ ] Define the highlights from release PRs as suggestion to be included on Blog Post
+- [ ] Talk to the Marketing Team about the Blog Post release
+- [ ] Talk to the Documentation Team about the Docs release
+- [ ] Sync translations from [LingoHub](https://translate.lingohub.com/rocketchat/rocket-dot-chat/dashboard)
+
+## Release Candidate 1 - On the 20th
+- [ ] Delete branch `release-candidate`
+- [ ] Create branch `release-candidate` based on `develop`
+- [ ] On branch `release-candidate` run `npm run release` and follow the steps
+- [ ] Publish the branch and the generated tag
+- [ ] Edit the tag on GitHub and paste the generated History removing the version from the first line and mark the checkbox **This is a pre-release**
+- [ ] Ensure the build is passing on CircleCI
+- [ ] Ensure the build is passing on Docker Hub
+
+<!-- Copy following block for next release candidates
+## Release Candidate {release-candidate-version}
+- [ ] Merge `develop` into `release-candidate` branch
+- [ ] On branch `release-candidate` run `npm run release` and follow the steps
+- [ ] Publish the branch and the generated tag
+- [ ] Edit the tag on GitHub and paste the generated History removing the version from the first line and mark the checkbox **This is a pre-release**
+- [ ] Ensure the build is passing on CircleCI
+- [ ] Ensure the build is passing on Docker Hub
+-->
+
+## Final Release - On the 27th
+- [ ] Merge `develop` into `release-candidate` branch
```

Comment:
I'm not sure about this

Reply:
@sampaiodiego Explain please.. :)

Reply:
sry, it was too late to explain 😛 

so, I might not have understood correctly at the time I read. but reading it again I guess we will **not** merge *anything* to `develop` during freeze time? if so, this sentence is correct

Reply:
That would be incorrect. We will be merging bug fixes into develop. Ideally between the time of the first release candidate and the release there will be more release candidates which contain the bug fixes merged, thus when the final release happens there won't be much to merge.

Reply:
ok, true, you're right.. but we'll be merging only bug fixes introduced by the release candidate in progress.

Reply:
Are we or are we not? I honestly feel that we do not need to limit ourselves to saying we will only merge bugs introduced by the release candidate. Otherwise we will be limiting ourselves and even be questioning whether the bug was introduced in the release candidate or whether it existed before.